src/internal/observable/dom/WebSocketSubject.ts

```diff
@@ -7,6 +7,7 @@ import { ReplaySubject } from '../../ReplaySubject';
 import { Observer, NextObserver } from '../../types';
 import { tryCatch } from '../../util/tryCatch';
 import { errorObject } from '../../util/errorObject';
+import { root } from '../../util/root';
```

Comment:
We want to get away from using `root`.

Reply:
@benlesh Updated this to remove `root` and just see if `WebSocket` exists in the global scope.  Is there a specific reason we're moving away from using `root`?

Reply:
It causes weird issues with some bundling setups, and it's just unnecessary code for the most part.